src/map/latent_effect.h

```diff
@@ -71,27 +71,27 @@ enum class LATENT : uint16
     JOB_MULTIPLE_AT_NIGHT  = 39, // PARAM: 0: ODD, 2: EVEN, 3-X: DIVISOR
     EQUIPPED_IN_SLOT       = 40, // When item is equipped in the specified slot (e.g. Dweomer Knife, Erlking's Sword, etc.) PARAM: slotID
     DURING_WS              = 41, // During WS
-    // 42 free to use
-    WEAPON_DRAWN_HP_UNDER = 43, // PARAM: HP PERCENT
-    NATION_CITIZEN        = 44, // Triggered by player being citizen of nation matching param: 0 San d'Oria, 1 Bastok, 2 Windurst
-    MP_UNDER_VISIBLE_GEAR = 45, // mp less than or equal to %, calculated using MP bonuses from visible gear only
-    HP_OVER_VISIBLE_GEAR  = 46, // hp more than or equal to %, calculated using HP bonuses from visible gear only
-    WEAPON_BROKEN         = 47, //
-    IN_DYNAMIS            = 48, //
-    FOOD_ACTIVE           = 49, // food effect (foodId) active - PARAM: FOOD ITEMID
-    JOB_LEVEL_BELOW       = 50, // PARAM: level
-    JOB_LEVEL_ABOVE       = 51, // PARAM: level
-    WEATHER_ELEMENT       = 52, // PARAM: 0: NONE, 1: FIRE, 2: ICE, 3: WIND 4: EARTH, 5: THUNDER, 6: WATER, 7: LIGHT, 8: DARK
-    NATION_CONTROL        = 53, // checks if player region is under nation's control - PARAM: 0: Under own nation's control, 1: Outside own nation's control
-    ZONE_HOME_NATION      = 54, // in zone and citizen of nation (aketons)
-    MP_OVER               = 55, // mp greater than # - PARAM: MP #
-    WEAPON_DRAWN_MP_OVER  = 56, // while weapon is drawn and mp greater than # - PARAM: MP #
-    ELEVEN_ROLL_ACTIVE    = 57, // corsair roll of 11 active
-    IN_ASSAULT            = 58, // is in an Instance battle in a TOAU zone
-    VS_ECOSYSTEM          = 59, // Vs. Specific Ecosystem ID (e.g. Vs. Plantoid: Accuracy+3)
-    VS_FAMILY             = 60, // Vs. Specific Family ID (e.g. Vs. Korrigan: Accuracy+3)
-    VS_SUPERFAMILY        = 61, // Vs. Specific SuperFamily ID (e.g. Vs. Mandragora: Accuracy+3)
-    MAINJOB               = 62, // mainjob - PARAM: JOBTYPE
+    WEATHER_CONDITION      = 42, // PARAM: 0: FAIR, 1: SUNSHINE, 2: CLOUDS, 3: FOG
```

Comment:
Comment should be "See Weather.h for WEATHER enum" since it's not just for those 4 params
also, not sure why there's an extra space for alignment here when it seems to be aligning to nothing?

Reply:
Updates comment 😊

Not sure I understand your note, "not sure why there's an extra space for alignment here when it seems to be aligning to nothing", prior to adding that space the enum was not aligned? I thought it was required to have enums `=` aligned to one another?

<img width="326" alt="Screenshot 2024-08-04 at 11 19 50 PM" src="https://github.com/user-attachments/assets/17ae3ca0-7d45-4e66-bfa4-496de28f2926">

Reply:
ah, I see what you did now. That wasn't visible on the diff but it is when I look at the code after the commit. nevermind there

Reply:
Ahh, gotcha. Thanks for the review!